.eslintignore

```diff
@@ -2,3 +2,4 @@
 app/**
 vendor/**
 config/webpack.config.js
+config/webpack.common.config.js
```

Comment:
I think we should just ignore `config/**`, no point linting any of it (since we lint with es6 rules etc and config has to be legacy)